airbyte-integrations/connectors/source-cockroachdb/src/test-integration/java/io/airbyte/integrations/source/cockroachdb/CockroachDbSourceDatatypeTest.java

```diff
@@ -315,25 +302,21 @@ protected void initTests() {
             .addExpectedValues("a", "abc", "Миші йдуть;", "櫻花分店", "", null, "\\xF0\\x9F\\x9A\\x80")
             .build());
 
-    // TODO https://github.com/airbytehq/airbyte/issues/4408
-    // JdbcUtils-> DATE_FORMAT is set as ""yyyy-MM-dd'T'HH:mm:ss'Z'"" for both Date and Time types.
-    // So Time only (04:05:06) would be represented like "1970-01-01T04:05:06Z" which is incorrect
```

Comment:
This comment is still relevant since the returned value still have 1970-01-01 in the full date string.

Reply:
Agreed. Returned back the comment, regarding the DateTime format. However, all java source connectors will send full DateTime in JSON for now. As we need standardization for all related DateTime formats.
Please, see:
https://github.com/airbytehq/airbyte/blob/d71871edc61c84d87aca57cfdadaa9dee36e9471/airbyte-db/lib/src/main/java/io/airbyte/db/DataTypeUtils.java#L20

So this behavior will be changed when we will sure all destination connectors are able to parse date and time formats.

Reply:
Got it. Make sense.